lib/heap_sort.rb

```diff
@@ -1,8 +1,24 @@
-
+require 'pry'
+require_relative 'min_heap'
 
 # This method uses a heap to sort an array.
-# Time Complexity:  ?
-# Space Complexity: ?
+# Time Complexity:  O(n) followed by O(log n) beacause going through entire array for first operation
```

Comment:
It's actually `O(n log n)` which is because you're adding `n` elements to the heap which each take `log n` time.  Then you remove `n` elements which takes `log n` time each.